src/containers/Teams/Resources/InternalBountyAccountRefills/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React, {FC} from 'react';
+
+import {DocContainer} from 'components';
+import 'containers/Teams/Resources/Resources.scss';
+
+const InternalBountyAccountRefills: FC = () => {
+  return (
+    <DocContainer className="Resource" title="Bounty Account Refills">
+      <p>
+        For any teams that use the bounty system, the related team lead will manage a bounty account. This will be an
+        account where coins will be sent from the governors. These coins will be used by the team lead for sending
+        reward payments when bounties are completed. These payments will also be monitored by the auditing team.
+      </p>
+      <p>
+        When the funds in a team's bounty account are depleted the team lead may request additional funding via a GitHub
+        issue in the{' '}
+        <a href="https://github.com/thenewboston-developers/Activity-Reports" target="_blank" rel="noreferrer">
```

Comment:
We can use the `<A>` component for this so we get this little icon.

<img width="284" alt="Screen Shot 2021-04-07 at 6 31 14 PM" src="https://user-images.githubusercontent.com/8547538/113954855-8298ab80-97cf-11eb-8771-cbe6f5d36bb7.png">

Reply:
yep done